docs/concepts/vega-chain.md

```diff
@@ -83,63 +96,123 @@ Exceptions to automatic nomination:
+There is no token slashing, i.e., a tokenholder cannot lose their tokens through any actions of as validator.
 
-* When staking **locked tokens**, the Vega node interacts with the ERC20 vesting contract, which holds locked tokens and provides the same utility as the staking bridge smart contract. 
+Read more: [How a validator node's performance is determined](/docs/concepts/vega-chain#validator-node-performance)
 
-Whether tokens are unlocked or locked, the bridge events make the Vega network of how many tokens a given party has associated and/or unassociated.
+## Validator nodes
+The Vega network is operated by a number of independent validators, who each run a node. Validator nodes are responsible for agreeing on the order of transactions and creating new blocks so that all nodes can agree on the state of the network. 
 
-All events (including the above, plus stake per validator and others) are only registered after a certain number of block confirmations, as defined by the network parameter `blockchains.ethereumConfig`. 
+If a validator's stake or performance is sub-par, their validator score will be lowered, and that validator's node will be chosen less frequently to propose a block. (This can also affect the rewards they and their nominators receive.) Vega feeds the voting power of each validator node to the Tendermint consensus algorithm. 
 
-:::note Further reading
-The staking bridge contracts can be found on the [Staking Bridge repository](https://github.com/vegaprotocol/Staking_Bridge) on GitHub.
+In this version of restricted mainnet, validators will not lose stake or rewards if they have a temporary interruption of service.
+
+### Validator node performance
+A validator node's performance is expressed through a validator score, and in the future, a performance score as well. 
+
+The validator score is calculated for each epoch, based on how much stake a validator has as well as other factors including the total number of validators and the optimal stake. 
+
+If, at the end of an epoch, a validator does not have sufficient stake self-nominated or has overall too much stake, then their validator score will be lowered, which can impact the rewards a validator and its' nominators receive. 
+
+Below are the two factors that can lower a validator's score, and why. 
+
+#### Not enough self-nominated stake
+Self-nominated stake refers to the amount of VEGA a validator has staked to their own node.  The minimum stake amount required is set using the network parameter `reward.staking.delegation.minimumValidatorStake`. Not having enough self-nominated stake can have an impact on rewards. 
+
+* **Network risk**: A validator who has not committed enough stake to meet the minimum is a risk to the network because they may not be invested in keeping the network running
+* **Validator score**: If a validator does not meet the `reward.staking.delegation.minimumValidatorStake`, then the validator's score is set to zero
```

Comment:
that's not yet true in 0.47.6. If that was correct neither the validator nor parties who nominated them wouldn't get a reward which is still not the case. The only sanction is that the validator itself isn't rewarded, but it's score is what it is based on its total stake.

Reply:
Ah, I see what you mean, both can't be true at the same time. I've changed to make it less specific